mappings/net/minecraft/client/gui/screen/achievement/AchievementWidget.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_594 net/minecraft/client/gui/screen/achievement/AchievementWidget
```

Comment:
`net.minecraft.client.toast.AchievementToast`, according to yarn.
`AdvancementWidget` is what's used to display advancements in the advancement tree, not in-game notification.